reputation/reputation_test.go

```diff
@@ -0,0 +1,662 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+package reputation
```

Comment:
so this is crazy, but make sure to have an empty whitespace between the copyright and the package definition. otherwise go will think the copyright is the documentation for package reputation